### DISCUSSION TITLE
Client RadSec - Common Name &/or Alt Name - Rewrite Block

### DISCUSSION THREAD

**Discussion by DSWall2021** [radsecproxy/radsecproxy#89] [Q&A] [answered]
Hi All,

Would anybody happen to know if it is possible to add the RadSec Client Cert Common Name &/or Cert Alt Name in the ReWrite Block e.g. addAttribute 30:$ClientCertCommonName, with many ReadSec clients (Access Points/Controllers) this would obviously be dynamic per client/connection?

Thanks in advance.

**Comment by fmauchle** [comment, marked as answer]
No, radsecproxy cannot use any attribute contents (other than the attribute the rewrite is applied to) or connection properties in rewrite.
